packages/blockchain-wallet-v4-frontend/src/modals/OrderMyCard/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import { compose } from 'redux'
+
+import Flyout, { duration } from 'components/Flyout'
+import { ModalName } from 'data/modals/types'
+import modalEnhancer from 'providers/ModalEnhancer'
+
+import { ModalPropsType } from '../types'
+import CreateCardStep from './CreateCardStep'
+
+const OrderMyCard = (props: Props) => {
```

Comment:
Could you destructure these props?

Reply:
In this case I need it like this because I have to pass all of them to Flyout component at line 29